code/modules/martial_arts/judo.dm

```diff
@@ -20,13 +20,30 @@
 	slot_flags = SLOT_FLAG_BELT
 	flags_2 = ALLOW_BELT_NO_JUMPSUIT_2
 	w_class = WEIGHT_CLASS_BULKY
+	storage_slots = 3
+	max_combined_w_class = 6
 	var/datum/martial_art/judo/style
+	can_hold = list(
+		/obj/item/grenade/flashbang,
+		/obj/item/grenade/chem_grenade/teargas,
+		/obj/item/reagent_containers/spray/pepper,
+		/obj/item/restraints/handcuffs,
+		/obj/item/flash,
+		/obj/item/food/snacks/donut,
+		/obj/item/flashlight/seclite,
+		/obj/item/holosign_creator/security,
+		/obj/item/holosign_creator/detective,
+		/obj/item/restraints/legcuffs/bola,
+		/obj/item/detective_scanner)
 
-/obj/item/judobelt/Initialize()
+/obj/item/storage/belt/judobelt/update_weight()
+	w_class = WEIGHT_CLASS_BULKY
```

Comment:
Is this needed when the belt is already bulky?

Reply:
This prevents the belt from being baggable once you empty it out the first time, else itll just inherit the `update_weight()` from Belts and suddenly be storeable (god i hate my own belt code).

Reply:
As discussed in discord the cleanup of `update_weight()` can be done in a separate PR